gits/test/capabilities_test.py

```diff
@@ -631,9 +631,11 @@ def test_cap_set_color_pair(self):
     def test_cap_set_color(self):
         pass
 
-    @unittest.skip('skip')
     def test_cap_sgr0(self):
```

Comment:
Put the test between `test_cap_sc` and `test_cap_smso` to arrange names in alphabetical order.